src/WilderMinds.RssSyndication.Tests/WilderMinds.RssSyndication.Tests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>RssSyndication.Tests Class Library</Description>
     <Authors>shawn</Authors>
-    <TargetFramework>net5.0</TargetFramework>
```

Comment:
.NET 5 is the LTS, I don't want to move this to 6 and cause issues for other users.  Do you think this is an issue?

Reply:
Since this is just hte test project, it might be fine. Updating package versions is fine, just don't want to stop the end-users from being able to use the library.

Reply:
I might be wrong, but I thought .NET 6 is the LTS and that .NET 5 has support level Current with end of support in just about a week from now (08.05.2022?). Am I misreading the [dotnet support policy](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core)?

For current end users, they would not be affected by this upgrade since it would not go into effect until the next version of the nuget? The class library is targeting [.netstandard 2.1](https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1) that is supported by both .NET 5 and .NET 6. Thus it should not break anything for current users relying on .NET 5.

Reply:
Hate when I remember it wrong ; )  I'll merge this now. This doesn't have any functionality changes so I won't push a new package. Thanks for the help!